### DISCUSSION TITLE
[Bug]: fresh install of docker nextcloud/all-in-one failes

### DISCUSSION THREAD

**Discussion by wiechers** [nextcloud/all-in-one#1972] [Questions] [answered]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Try to install latest docker image of nextcloud/all-in-one either via cli or portainer w/ and w/o official compose file https://github.com/nextcloud/all-in-one/blob/main/docker-compose.yml fails. I do not want to use an official domain name (internal usage only) nor do I want to use borg backup in favioure of my own solution. Disable BORG via env DISABLE_BACKUP_SECTION=true does not work. After first login I'm asked to fill out my domain and the path/passwd to the bork backup which does, of cause, not exists. There is no option to skip this stuff and continue with the rest of the setup. Stuck!

### Steps to reproduce

Try install of nextcloud/all-in-one:lastest on fresh installed debian w/o an previous nextcloud configs/volumes/... 

### Expected behavior

Initial setup screen

### Installation method

Official All-in-One appliance

### Operating system

Debian/Ubuntu

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
none
```

### List of activated Apps

```shell
none
```

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/local-instance.md if you want to run AIO locally
